catalog/video/video-security/src/main/java/org/codice/alliance/video/security/claims/videographer/VideographerClaimsHandler.java

```diff
@@ -100,19 +100,21 @@ public ProcessedClaimCollection retrieveClaimValues(ClaimCollection claims,
             ClaimsParameters parameters) {
         ProcessedClaimCollection claimsColl = new ProcessedClaimCollection();
         Principal principal = parameters.getPrincipal();
-        for (Claim claim : claims) {
-            URI claimType = claim.getClaimType();
-            List<String> value = claimsMap.get(claimType);
-            if (value != null) {
-                ProcessedClaim c = new ProcessedClaim();
-                c.setClaimType(claimType);
-                c.setPrincipal(principal);
-                value.forEach(c::addValue);
-                claimsColl.add(c);
-            }
-        }
 
         if (principal instanceof VideographerPrincipal) {
+
+            for (Claim claim : claims) {
+                URI claimType = claim.getClaimType();
+                List<String> value = claimsMap.get(claimType);
+                if (value != null) {
```

Comment:
Good point. Since the only way `value` could be `null` is if `claimType` is `null`, do you think we should change the condition to be `if (claimtype != null)` instead?